src/pages/Gallery/Gallery.tsx

```diff
@@ -1,7 +1,118 @@
-import React from "react"
+import React, { useState } from "react"
+
+import styles from "./Gallery.module.scss"
+import classnames from "classnames"
+
+const Lines = () => {
+  return (
+    <div className={styles.lines}>
+      {Array.from({ length: 6 }).map((_, index) => (
+        <div key={index} className={styles.line} />
+      ))}
+    </div>
+  )
+}
 
 const Gallery = () => {
-  return <h1>Wojak gallery here</h1>
+  const [isFilterOpen, setIsFilterOpen] = useState(false)
+
+  return (
+    <>
+      <div>
+        <div className={styles.pageContainer}>
+          <div className={styles.galleryContainer}>
+            <div className={styles.headerContainer}>
+              <div>
+                <div className={styles.filterDropdownContainer}>
+                  <div>
+                    <button
+                      type="button"
+                      onClick={() => {
+                        setIsFilterOpen(!isFilterOpen)
+                      }}
+                      className={styles.dropdownButton}
+                      id="options-menu"
+                      aria-expanded="true"
+                      aria-haspopup="true"
+                    >
+                      Filter by trait
+                      <svg
+                        className={styles.arrowIcon}
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 20 20"
+                        fill="currentColor"
+                        aria-hidden="true"
+                      >
+                        <path
+                          fillRule="evenodd"
+                          d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
+                          clipRule="evenodd"
+                        />
+                      </svg>
+                    </button>
+                  </div>
+                  <div
+                    className={classnames(
+                      styles.dropdownContentContainer,
+                      styles.transform,
+                      isFilterOpen ? styles.dropdownOpen : styles.dropdownClose,
+                    )}
+                    role="menu"
+                    aria-orientation="vertical"
+                    aria-labelledby="filter-menu"
+                  >
+                    <div className={styles.dropdownContent} role="none">
+                      <button className={styles.dropdownContentItem} role="menuitem">
+                        Trait 1
+                      </button>
+                    </div>
+                  </div>
+                </div>
+
+                <div className={styles.counterContainer}>
```

Comment:
I don't think this is button, is something like "informative box", if you have 0 wojaks, there should be "painy" one wojak image inside :D

Reply:
ESLint complained about my a tag and since I did not have a proper href="#" I did what ESLint suggested, which is why I converted it into a button.